10_fibonacci/solution/fibonacci-solution.js

```diff
@@ -1,10 +1,19 @@
 const fibonacci = function(count) {
-	if (count < 0) return "OOPS"
-	const fibPart = [0, 1];
-	for (let index = 1; index < count; index++) {
-		fibPart.push(fibPart[index] + fibPart[index -1]);
-	}
-	return fibPart[count];
+    if (count < 0) return "OOPS";
+    if (count === 0) return 0;
+
+    let firstPrev = 1;
+    let secondPrev = 0;
+
+    // For clarification: curr stands for current. This is standard syntax
+    
+    for (let i = 2; i <= count; i++) {
+        let curr = firstPrev + secondPrev;
+        secondPrev = firstPrev;
+        firstPrev = curr;
```

Comment:
Just one more small change please. Can we go for `current` instead of `curr` and remove the comment? This exercise is fairly early in the curriculum and I want the variable names to be as clear as possible for the learners. Thanks!

Reply:
Done